src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -17,20 +21,20 @@
           ref="qrCodeVideo"
         />
       </div>
-      <div v-if="!cameraAllowed">
-        {{ $t('modals.qrCodeReader.cameraNotAllowed') }}
+      <div v-if="!cameraAllowed" class="text">
+        <div v-html="$t('modals.qrCodeReader.cameraNotAllowed')" />
       </div>
     </div>
-
     <template
       v-if="!cameraAllowed"
       slot="footer"
     >
-      <Button @click="cancelReading">
+      <Button @click="cancelReading" class="button">
```

Comment:
```suggestion
      <Button @click="cancelReading">
```
`Button` component already have `.button` class.